nestkernel/connection_manager.cpp

```diff
@@ -805,7 +805,6 @@ nest::ConnectionManager::divergent_connect( index source_id,
 
       // here we fill a parameter dictionary with the values of the current loop
       // index.
-      par_i->clear();
```

Comment:
In the loop following on lines 808-814, we always fill all entries in the dictionary with new values (on first pass, the dict is empty). Therefore, it is not necessary to explicitly clear the dict.